### ISSUE TITLE
Change "user" to "visitor" in all key metric tiles

### ISSUE THREAD

**Issue by nfmohit** [google/site-kit-wp#7641]
## Feature Description

In an effort to improve the language in Site Kit to be more humanly understandable, all Site Kit key metric tiles (including News key metric tiles) should be updated so that any instance of the word "user" is changed to "visitor".

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by marrrmarrr** [comment]
Sure, let's repace these for consistency. 
For the `Visit length` tooltip "Average duration (in seconds) of visitors' visits" does sound a bit repetitive, so let's shorten this to "Average visit duration (in seconds)".
